libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -421,15 +465,57 @@ bool AP_Arming::battery_checks(bool report)
         }
 
         for (uint8_t i = 0; i < _battery.num_instances(); i++) {
+            // voltage check
             if ((_min_voltage[i] > 0.0f) && (_battery.voltage(i) < _min_voltage[i])) {
                 if (report) {
-                    gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Battery %d voltage %.1f below minimum %.1f",
+                    gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Battery %u voltage %.1f below min %.1f",
```

Comment:
Why change this? minimum is the correct word, and if you format out and actual battery this doesn't overflow the status text length. (Unless you are looking at 100V+ systems)

Reply:
I adapted to other wordings. I thought that it would be better for you to talk words together. However, I understand the idea that if you do not need to omit the word, you will use it as it is. Therefore, this process is restored.